.travis.yml

```diff
@@ -66,6 +66,23 @@ jobs:
               tags: true
               condition: "$TRAVIS_OS_NAME = 'linux'"
 
+    - name: "Linux - Docker - Deploy - Stable"
+      stage: deploy
+      os: linux
+      before_install: skip
+      install: skip
+      script:
+        - wget https://raw.githubusercontent.com/plume-lib/trigger-travis/master/trigger-travis.sh
+      deploy:
+        - provider: script
+          script: bash trigger-travis.sh --pro Drakkar-Software OctoBot-Builder $TRAVIS_TOKEN
+          cleanup: false
+          on:
+            repo: "$GH_REPO"
+            branch: "$DEPLOY_BRANCH"
+            tags: true
+            condition: "$TRAVIS_OS_NAME = 'linux'"
```

Comment:
👍

Reply:
Should we also use this for "Linux - Python 3 nightly - Deploy" ?

Reply:
I don't understand the question

Reply:
My bad, I misunderstood the idea